packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../application_package.dart';
+import '../build_configuration.dart';
+import '../globals.dart';
+import '../ios/mac.dart';
+import '../runner/flutter_command.dart';
+
+class BuildIOSCommand extends FlutterCommand {
+  BuildIOSCommand() {
+    argParser.addFlag('simulator', help: 'Build for the iOS simulator instead of the device');
```

Comment:
perhaps suffix with ` (macos ony)`; we have a few other commands that have these suffixes